lib/workos/organizations/organizations.ex

```diff
@@ -94,7 +94,7 @@ defmodule WorkOS.Organizations do
   WorkOS.Organizations.list_organizations()
   """
   def list_organizations(params \\ %{}, opts \\ []) do
-    query = process_params(params, [:domains, :limit, :before, :after, :order])
-    get("/organizations", query, opts)
+    body = process_params(params, [:domains, :limit, :before, :after, :order])
```

Comment:
Sorry, for gets, it makes sense to keep this `query`, as they'll be added as query parameters. It's just with PUT and POST requests, they'll be added as the `body`.